app/assets/javascripts/discourse/tests/fixtures/search-fixtures.js

```diff
@@ -1,823 +1,63 @@
 export default {
   "/search.json": {
-    users: [
```

Comment:
This search fixture is stale, the payload doesn't match what the server is returning anymore.